docs/understanding-tracking-design/introduction-to-tracking-design/best-practices/index.md

```diff
@@ -0,0 +1,252 @@
+---
+title: "Tracking design best practices"
+---
+
+Designing the events and entities to be captured in your tracking is a challenging task.
+This page aims to give guidance on how to approach it based on our experience.
+
+Overall, it suggests taking the following steps:
+
+1. Analyze the business use case and define the reports to be produced.
+2. Check our out-of-the-box data products to see if you can reuse or build on top of them.
+3. Choose the naming convention to follow in your schemas.
+4. Define your entities first.
+5. Introduce events as interactions of the entities.
+
+## Analyze your business use case and define reports
+
+Good tracking design comes from an understanding of the business use case.
+Identifying and analyzing the use case should be the first step before we design our tracking.
+This involves identifying the business outcomes that you want to achieve (e.g., acquire new customers, increase the number of signups, minimize abandoned carts)?
+It can often be answered by interviewing stakeholders who will consume the data in your organization.
+
+One of the ways to think about this is to sketch the reports that we want to have – the derived tables that the modeling process will extract from the raw events.
+Based on the reports, one can identify the entities, events, and SQL queries to be performed on top of them.
+
+:::tip Example – abandoned carts in e-commerce
+
+For example, if one of the use cases is to identify abandoned carts, we may want the following reports to explore the data:
+
+* What is the frequency of users abandoning carts for different segments of users?
+* What are the most commonly abandoned products in carts?
+* What leads users to return to the store and continue their purchase?
+
+We need to go deeper into specifying what exactly the derived tables for these reports should contain.
+However, we can already see a few reusable entities that we need to track: user, cart, product, page referrer.
+The reports can also tell us what events we'll need as interactions of these entities: add product to cart, checkout step, transaction, page view.
+
+:::
+
+## Make use of our out-of-the-box schemas
+
+Snowplow trackers and data models provide a rich set of events and entities to cover common use cases.
```

Comment:
Maybe in this section, maybe another, but something to call out check what they are already tracking/have defined - especially for entities where they should re-use them as much as possible and might only need to extend an existing thing rather than make a whole new one?